sdk/identity/azure-identity/src/main/java/com/azure/identity/DefaultAzureCredentialBuilder.java

```diff
@@ -23,6 +34,57 @@ public DefaultAzureCredentialBuilder authorityHost(String authorityHost) {
         return this;
     }
 
+
+    /**
+     * Specifies whether the {@link EnvironmentCredential} will be excluded from the {@link DefaultAzureCredential}
+     * authentication flow. Setting to true disables reading authentication details from the process' environment
+     * variables.
+     *
+     * @param shouldExclude the status indicating to exclude the credential or not.
+     * @return An updated instance of this builder with the Environment credential exclusion set as specified.
+     */
+    public DefaultAzureCredentialBuilder excludeEnvironmentCredential(boolean shouldExclude) {
```

Comment:
Do you think that we need the boolean parameter? Should builder.excludeEnvironmentCredential() just set excludeEnvironmentCredential to true?

Reply:
Yeah, agreed and updated.
Just curious, in .NET I didn't see this pattern though.

Reply:
in .NET we don't have builders or setters ;)